apps/prairielearn/assets/scripts/instructorAiGenerateDraftsSampleQuestionClient.ts

```
@@ -0,0 +1,827 @@
+
+        if (examplePrompt.answerType === 'number') {
+          // Add relative and absolute tolerance if available
+          {
+            if (examplePrompt.rtol && examplePrompt.atol) {
+              placeholderText = `${placeholderText} (rtol=${examplePrompt.rtol}, atol=${examplePrompt.atol})`;
+            } else if (examplePrompt.rtol) {
+              placeholderText = `${placeholderText} (rtol=${examplePrompt.rtol})`;
+            } else if (examplePrompt.atol) {
+              placeholderText = `${placeholderText} (atol=${examplePrompt.atol})`;
+            }
+          }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)